src/lib/output/renderer.ts

```diff
@@ -201,7 +201,24 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            let filename = Path.join(path, 'theme.js');
+            if (!FS.existsSync(filename)) {
+                this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            } else {
+                try {
+                    const plugin = require(filename);
+                    if(typeof plugin === 'function') {
+                        plugin(this.application);
```

Comment:
It doesn't break any functionality though, it makes it consistent. What you have is not consistent, the behaviour is different depending on how you are exporting the class/function.